Rust/rust/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[workspace]
+
+members = [
+    "Samples/allapis",
+    "Samples/ADC/adc_high_level_app",
+    "Samples/DHCP",
+    "Samples/CertStore/cert_store_high_level_app",
+    "Samples/CustomNTP/custom_ntp_high_level_app",
+    "Samples/GPIO/gpio_high_level_app",
+    "Samples/HelloWorld/hello_world_high_level_app"
```

Comment:
Does `/Samples/*` work?

Reply:
No it doesn't work... it doesn't cope with samples/ADC/adc_high_level_object, where the sample app is nested an extra level down.